### DISCUSSION TITLE
The sound control does not go with MacBook Pro M1Pro + HP 27ea monitor

### DISCUSSION THREAD

**Discussion by pyroranking** [MonitorControl/MonitorControl#858] [Q&A]
Hi. The sound control keys does not work (nor slider) with **MacBook Pro M1-Pro** in a **HP 27ea** monitor screen... (macOS **Monterey 12.1** + **MonitorControl 4.0.2** Compilation 6965). The sound symbols (increase/decrease and mute) appear on the screen but they do not control the external speakers of the monitor.
The brightness keys works fine (thank you!).
What could be the solution? Thank you in advance!

**Comment by waydabber** [comment]
Unfortunately on some HP displays control of external speakers do not work. This come up in the past, there is no known solution to this as it is a firmware issue with the display. :(

**Reply by pyroranking** [reply]
Ops! what a pity so! 😢 Thank you very much anyway waydabber. What other companies displays have this issues? (I'm searching a new display for my parents home I would not have this problem again). Is there any list of companies/displays that have this kind of problems with MonitorControl? thank you! 😉
